pages/shared/studentGroupAssessmentHelpers/studentGroupAssessmentHelpers.js

```diff
@@ -0,0 +1,137 @@
+var ERR = require('async-stacktrace');
```

Comment:
Let's move this file (`studentGroupAssessmentHelpers.js`) and the corresponding `.sql` into `lib`, since that's where this kind of shared code generally goes. I'd suggest `lib/groups.js`/`lib/groups.sql`.

Reply:
Done in [05aacad](https://github.com/PrairieLearn/PrairieLearn/pull/5290/commits/05aacadec3bdb06cbca81edddd52c0820e9e71d8)